source/vibe/data/json.d

```diff
@@ -44,7 +44,7 @@ unittest {
 public import vibe.data.serialization;
 
 public import std.json : JSONException;
-import std.algorithm : equal, min;
+import std.algorithm : equal, min, max;
```

Comment:
Minor: I would suggest that we reduce, and even remove, any selective import at the top. Until 313 and 314 are fixed, those cause symbols leakage, and upgrading a version of Vibe.d which removes one of the import might break code. And when 313 / 314 are fixed, it'll definitely breaks a lot of code...